### PR TITLE
Sup 2409 mobile dashboard

### PR DESCRIPTION
**Pull request by nlitwin**
Merging to test in dev on iPhone. @parthshah @vikasrohit @tladendo Let me know if you see any problems.

### CODE REVIEW

File: app/layout/footer/footer.jade [hunk 1]
```diff
@@ -2,20 +2,25 @@ footer.bottom-footer
...
     .menu-item-header.show-small OTHERS
+
     ul.submenu
-        li.submenu-item #[a.menu-link(ng-click="$event.stopPropagation();" href="/sitemap") SITEMAP]
-        li.submenu-item #[a.menu-link(ng-click="$event.stopPropagation();" href="/about") ABOUT US]
-        li.submenu-item #[a.menu-link(ng-click="$event.stopPropagation();" href="/contact-us") CONTACT US]
-        li.submenu-item #[a.menu-link(ng-click="$event.stopPropagation();" href="https://help.{{domain}}" target="_blank") HELP CENTER]
-        li.submenu-item #[a.menu-link(ng-click="$event.stopPropagation();" href="https://www.{{domain}}/community/how-it-works/privacy-policy/") PRIVACY POLICY]
-        li.submenu-item #[a.menu-link(ng-click="$event.stopPropagation();" href="https://www.{{domain}}/community/how-it-works/terms/") TERMS]
+      li.submenu-item #[a.menu-link(ng-href="https://www.{{domain}}/sitemap") SITE MAP]
```
Comment (nlitwin) [inline app/layout/footer/footer.jade]: Makes links work locally and per environment

File: assets/css/layout/footer.scss [hunk 2]
```diff
@@ -71,6 +71,19 @@
 			height: 27px;
 		}
 	}
+
+  .bottom-footer nav {
```
Comment (nlitwin) [inline assets/css/layout/footer.scss]: Moves the footer links up to connect with the mobile nav links.

File: assets/css/layout/header.scss [hunk 3]
```diff
@@ -148,17 +148,18 @@
 
 .search-wrapper {
   input {
-    font-weight: bold;
-    border: 1px solid #7f7f7f;
-    color: #fff;
-    border-radius: 4px;
     display: block;
+    box-sizing: border-box;
```
Comment (nlitwin) [inline assets/css/layout/header.scss]: border-box fixes the search bar going off the screen

File: app/sitemap/sitemap.jade [hunk 4]
```diff
@@ -1,58 +1,58 @@
...
+        li 
+          a(href="/blog") Blog
+          
+    section.sitemap-nav.hide-small
+      h2.sitemap-header My Account
+      ul
+        li 
+          a(href="/my-dashboard") Dashboard
+        li 
+          a(href="/members/") My Profile
```
Comment (parthshah) [inline app/sitemap/sitemap.jade]: This link is in correct? Shouldn't this be dynamic?

File: app/sitemap/sitemap.jade [hunk 5]
```diff
@@ -1,58 +1,58 @@
...
+        li  
+          a(href="/settings/profile") Settings
```
Comment (parthshah) [inline app/sitemap/sitemap.jade]: Shouldn't this be "ui-sref"?

File: app/sitemap/sitemap.jade (hunk 4 above)
Comment (nlitwin) [inline app/sitemap/sitemap.jade]: I didn't make any changes to this file except adding a space between SITE and MAP, the rest is improper indenting that Sublime Text auto fixed. I haven't taken a look at the site map code - but I can make the necessary updates if we want to change it.

Comment (parthshah) [inline app/sitemap/sitemap.jade]: Please fix these. These are minor issues so hopefully it doesn't take too
long.
Thanks


On Mon, Nov 9, 2015 at 12:36 PM, Nicholas Litwin <notifications@github.com>
wrote:

> In app/sitemap/sitemap.jade
> <https://github.com/appirio-tech/topcoder-app/pull/543#discussion_r44327588>
> :
>
> > +        li
> > +          a(href="//apps.topcoder.com/forums") Forums
> > +        li
> > +          a(href="/community/statistics") Statistics
> > +        li
> > +          a(href="/community/events") Events
> > +        li
> > +          a(href="/blog") Blog
> > +
> > +    section.sitemap-nav.hide-small
> > +      h2.sitemap-header My Account
> > +      ul
> > +        li
> > +          a(href="/my-dashboard") Dashboard
> > +        li
> > +          a(href="/members/") My Profile
>
> I didn't make any changes to this file except adding a space between SITE
> and MAP, the rest is improper indenting that Sublime Text auto fixed. I
> haven't taken a look at the site map code - but I can make the necessary
> updates if we want to change it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appirio-tech/topcoder-app/pull/543/files#r44327588>.
>



-- 
Thanks
Parth Shah

**Pull request merged**